src/bin/dual-iir.rs

```diff
@@ -73,8 +75,39 @@ const SAMPLE_TICKS: u32 = 1 << SAMPLE_TICKS_LOG2;
 const SAMPLE_PERIOD: f32 =
     SAMPLE_TICKS as f32 * hardware::design_parameters::TIMER_PERIOD;
 
-#[derive(Clone, Copy, Debug, Tree)]
-pub struct Settings {
+#[derive(Clone, Debug, Tree)]
+pub struct FlashSettings {
+    #[tree(depth(3))]
+    pub dual_iir: RuntimeSettings,
+
+    #[tree(depth(1))]
+    pub net: NetSettings,
+}
+
+impl stabilizer::settings::AppSettings for FlashSettings {
+    fn new(net: NetSettings) -> Self {
```

Comment:
Doesn't work.

Reply:
Yeah the reason for the `new()` method here is to allow C::new() to be called from `stabilizer::hardware::setup()`. Otherwise, it necessitates a bit of boilerplate that is copy-pasted among the various applications.